modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Captcha/Captcha.es.js

```diff
@@ -46,6 +46,29 @@ const Captcha = ({html, name}) => {
 		}
 	}, [elRef]);
 
+	useEffect(() => {
+		function setNewCaptcha() {
+			const captcha = elRef.current.querySelector('img');
+			const url = new URL(captcha.src);
+
+			url.searchParams.set('t', Date.now());
+
+			captcha.setAttribute('src', url.href);
+		}
+
+		if (elRef.current) {
+			const refreshCaptcha = elRef.current.querySelector('a');
+
+			if (refreshCaptcha) {
+				refreshCaptcha.addEventListener('click', setNewCaptcha);
+
+				return () => {
+					refreshCaptcha.removeEventListener('click', setNewCaptcha);
+				};
+			}
+		}
+	}, [elRef]);
```

Comment:
@brunobasto I reversed the changes in `simplecaptcha.jsp` and ended up adding the same implementation here. I had problems with the previous implementation with Senna.js in navigation and also with the form portlet.